config/setupTests.js

```diff
@@ -32,6 +32,6 @@ global.innerWidth = 1080;
 
 Element.prototype.scrollTo = () => {};
 
-global.mockApi = (data, timeout = 60) =>
+global.mockApi = (data, timeout = 200) =>
   jest.fn().mockImplementation(() => new Promise((resolve) => setTimeout(() => resolve(data), timeout)));
-global.mockApiError = (data, timeout = 60) => jest.fn(() => new Promise((_, reject) => setTimeout(() => reject(data), timeout)));
+global.mockApiError = (data, timeout = 200) => jest.fn(() => new Promise((_, reject) => setTimeout(() => reject(data), timeout)));
```

Comment:
Any specific reason for increasing the timout time? It will slow down the test run.

Reply:
This has to be long enough to not be too fast in slower env.

It is just a temporary solution, in the next step I plan to refactor it so the promise would be triggered manually (no timeout) but that would be done separately. See https://issues.redhat.com/browse/RHCLOUD-18835